spec/requests/profile_preview_cards_spec.rb

```diff
@@ -118,10 +122,14 @@
         get profile_preview_card_path(user), as: :json
 
         preview_card = response.parsed_body
+        labels_to_attrs = %w[Work Education].index_with do |label|
```

Comment:
Since this is exactly the same block as above, would it make sense to move it to a `let` binding?